server/cli/src/refresh_dates.rs

```diff
@@ -43,23 +43,10 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("sensor", "last_connection_datetime"),
         ("stocktake", "created_datetime"),
         ("stocktake", "finalised_datetime"),
-        ("sync_log", "started_datetime"),
```

Comment:
These are also in get_exclude_timestamp_fields, not needed here (don't need to advance these ones as part of refresh_dates)